src/app/InteractionModelEngine.h

```diff
@@ -48,8 +48,8 @@
 #include <app/util/basic-types.h>
 
 #define CHIP_MAX_NUM_COMMAND_HANDLER 1
-#define CHIP_MAX_NUM_COMMAND_SENDER 1
-#define CHIP_MAX_NUM_READ_CLIENT 1
+#define CHIP_MAX_NUM_COMMAND_SENDER 64
```

Comment:
If they need to be the same, shouldn't we reference `kNumMaxActiveDevices`?  Else, we'll be forced to keep tweaking these to stay in sync.  Also, eventually people are going to want to have a build config for kNumMaxActiveDevices.  When such a thing is introduced, other dependent constants should be adjusted accordingly.